easybuild/easyconfigs/n/ncurses/ncurses-5.9_configure_darwin.patch

```diff
@@ -1,3 +1,5 @@
+# in relation to -no-cpp-precomp bug: https://lists.gnu.org/archive/html/bug-ncurses/2012-12/msg00001.html
+# by Fotis
```

Comment:
tried/proposed by Fotis, meant for MacOSX compatible nurses builds...

overall comment for this PR: we sorely need next-generation easyconfigs to consolidate the information!